### ISSUE TITLE
Data UI back end upgrade to match splash pages

### ISSUE THREAD

**Issue by holbrooa** [lzim/teampsd#430]
**Describe the opportunity**
Integration and translation of splash pages' data model conceptualization for count aggregation differentiation (e.g. the distinct summation computation), visualization standardization, and propagation automation.

**FEATURE**

**7. How would you like this feature to work?**
Improve charts and tables to use the PowerPivot instead of regular pivots, allowing unique counts, among other things.

**8. How urgent is this request? (Urgent/Not Urgent)**
Not Urgent

**9. Why is this feature important to you?**
The better pivots are better for a variety of reasons, including lining up with the splash charts.

**10. Is your feature request related to a problem? Please describe.**
Nah.

**11. Is there a deadline or other scheduled constraint driving your request?"**
Nah, but this has to be done before the data ui gets propagated.

**12. Additional context**
This shouldn't change the look and feel of the front end for users, except perhaps in very minor ways, but it is a big back-end change.

**STOP HERE. INFORMATION BELOW IS TO BE FILLED OUT BY WORKGROUPS AS PART OF TRIAGE AND WORK BREAKDOWN PROCESS**

**Estimated person hours to complete:**
12

**Key people (use @ assignment for people whose input will be necessary):**
@holbrooa 

**Key dependencies:**
Are there other tasks that need to be completed first?
No

**Constraints:**
Is there a resource constraint that may affect the development of the feature?
No

**Gantt chart needed? Attach link and/or screenshot.**
This is already on Savet's big one

**Comment by holbrooa** [comment]
New features were presented and decisions were made on 8/12 at https://meet.lucidmeetings.com/meeting/223386 and those decisions have now been implemented. See the meeting notes for the changes and decisions.

**Comment by lzim** [comment]
@staceypark and @holbrooa Do we have this adequately documented in other issues?
Should this issue be closed?

**Comment by holbrooa** [comment]
Yeah, let's close this one. There is only one idea missing at this point, the propagation code, but I think that should be a separate issue: https://github.com/lzim/teampsd/issues/594

**Comment by lzim** [comment]
Great, thanks @holbrooa
